src/components/modal/v2/parts/views/NI/Content.jsx

```diff
@@ -0,0 +1,71 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useState, useRef } from 'preact/hooks';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import ProductListLink from '../../ProductListLink';
+import Instructions from '../../Instructions';
+import styles from './styles/index.scss';
+import { useApplyNow } from '../../../lib';
+
+export const NI = ({ instructions, terms, buttonText, disclaimer, footer, listLink, contentBodyRef }) => {
+    const buttonRef = useRef();
+    const handleApplyNowClick = useApplyNow('Apply Now');
+    const [expandedState] = useState(false);
+
+    return (
```

Comment:
```suggestion
    const renderProductListLink = () => {
        if (useServerData()?.views?.length > 1) {
            return <ProductListLink>{linkToProductList}</ProductListLink>;
        }
        return <Fragment />;
    };

    return (
```
Suggesting this change here since I think it would be best to rename this ```linkToProductList``` to match Pay Monthly and Pi4 as closely as possible for similar things content-schema-wise. 

Also we can add a check here above the return that determines whether or not we should render the Product List Link component. If we have only one view returned (i.e. merchant is only eligible for just Pay in 4 or just NI), they won't see the link to the product list modal since they only get one product back.